crates/ra_assists/src/ast_editor.rs

```diff
@@ -49,7 +49,7 @@ impl<N: AstNode> AstEditor<N> {
 
     fn do_make_multiline(&mut self) {
         let l_curly =
-            match self.ast().syntax().children_with_tokens().find(|it| it.kind() == L_CURLY) {
+            match self.ast().syntax().children_with_tokens().find(|it| it.kind() == T!['{']) {
```

Comment:
Hm, interesting, I've missed that while reviewing the original PR.

Indeed, we need quotes here, because `{` break token tree structure. The API looks to sigil heavy as a result though. 

Let's maybe scale this back, remove support for {, }, (, ), [, ] from `T` and stick with `L_CURLY` and friends?